doc/sphinx/source/core/quantization.rst

```diff
@@ -0,0 +1,163 @@
+.. _quantization:
+
+Quantization
+============
+
+nGraph Quantization 
+-------------------
+
+Quantization refers the process of reducing the number of bits that represent a
+number. In the context of deep learning, weights and activations can be
+represented using 8-bit integers (or INT8) to compress the model size of a
+trained neural network, without any significant loss in model accuracy. Compared
+with 32-bit floating point (FP32), using arithmetic with lower precision such as
+INT8 to calculate weights and activation requires less memory. However, memory
+reduction varies based on the model (ResNet 50, Inception v3, BERT, etc.).
+
+Implementing a quantized model with nGraph
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To implement a quantized model with
+nGraph, provide a partially (or fully) quantized FP32 model (e.g., where the
+convolution layer in the model is replaced with a quantized convolution) to the
+nGraph library along with quantized parameters: weights, activations, scale, and
+zero point. 
+
+.. Note:: Currently, nGraph only supports quantization for inference.
+
+nGraph Quantized Operators (Ops)
+--------------------------------
+
+nGraph uses scale and zero point (also used by ONNX) to map real values to
+quantized values.  All quantized ops (APIs) in nGraph use scale and zero point
+and they can be used just like any other nGraph op. 
+
+**Scale**: the quantization scale for the tensor 
+
+**Zero point**: the zero point of the tensor 
+
+**Round mode**: used in combination with scale and zero point to round real 
+values to quantized values
+
+.. table:: nGraph Quantized Ops
+
+
+	+----------------------+-----------------------------------------------+
+	| Operator             | Description                                   |
+	+======================+===============================================+
+	| Quantize             | Maps real values (r) to quantized values (q)  |
+	|                      | using scale (s), zero point (z),              |
+	|                      | and round mode; produces a quantized tensor   |
+	+----------------------+-----------------------------------------------+
+	| Dequantize           | Maps quantized values (q) to real values (r)  |
+	|                      | using scale (s) and zero point (z); converts  |
+	|                      | a quantized tensor to a floating point tensor |
+	+----------------------+-----------------------------------------------+
+	| FakeQuantize         | Performs element-wise linear quantization     |
+	+----------------------+-----------------------------------------------+
+	| QuantizedConvolution | Performs quantized convolution; takes the     |
```

Comment:
The API is different for Convolution and QuantizedConvolution. So I dont think so we can say `takes the same parameters as the non-quantized convolution operator`

Reply:
Ashok to comment.

Reply:
i think the wording is misleading, we were meaning to say the computation is same (thus the data/inputs/params it operates on is similar), except for the data type it operates on (f32 vs int8).  @evelynfay perhaps we can remove this sentence (or clarify).

Reply:
Suggestion: 

Performs quantized convolution; the computation is the same as the non-quantized convolution operator, except the data type it operates on is INT8.

Reply:
Could we just use "Performs quantized convolution". That extra statement is kind of true, but lacks minute nuances.

Reply:
Recall that the reason we added the extra statement was to provide something useful for the reader. Per Bethany's feedback, "Performs quantized convolution" is just stating the obvious. Can you suggest a more nuanced statement?

Reply:
`Performs 8 bit convolution` ?